corehq/apps/hqwebapp/static/hqwebapp/js/multiselect_utils.js

```diff
@@ -54,10 +54,11 @@ hqDefine('hqwebapp/js/multiselect_utils', [
     };
 
     multiselect_utils.createFullMultiselectWidget = function (elementOrId, properties) {
-        assertProperties.assert(properties, [], ['selectableHeaderTitle', 'selectedHeaderTitle', 'searchItemTitle']);
+        assertProperties.assert(properties, [], ['selectableHeaderTitle', 'selectedHeaderTitle', 'searchItemTitle', 'willSelectAllListener']);
```

Comment:
What would you think of documenting these, either as a function comment or a file-level comment?

Reply:
See fbc681ac645d9d1d5a2048db5f7d17aceaf32802